File: final_dataset_A/generated_code/chatgpt/python/customer_information_endpoint-chatgpt.py
from flask import Flask, jsonify

app = Flask(__name__)

@app.route("/customers/<int:customer_id>", methods=["GET"])
def get_customer(customer_id):
    # Simulated customer data
    customer = {
        "customerId": customer_id,
        "firstName": "John",
        "lastName": "Doe",
        "email": "john.doe@example.com",
        "phone": "+1-555-123-4567",
        "status": "Active"
    }

    return jsonify(customer), 200

if __name__ == "__main__":
    app.run(debug=True)

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/CustomerInformationEndpoint-chatgpt.java
package com.example.demo.controller;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.*;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/api/customers")
public class CustomerController {

    @GetMapping("/{id}")
    public ResponseEntity<Map<String, Object>> getCustomer(
            @PathVariable Long id) {

        Map<String, Object> customer = new HashMap<>();
        customer.put("id", id);
        customer.put("firstName", "John");
        customer.put("lastName", "Doe");
        customer.put("email", "john.doe@example.com");
        customer.put("phone", "+1-555-123-4567");
        customer.put("status", "Active");

        return ResponseEntity.ok(customer);
    }
}